src/index.html

```diff
@@ -1,14 +1,60 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Moyo header</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Moyo header</h1>
-</body>
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link rel="stylesheet" href="./style.css">
+    <link
+    rel="preconnect"
+    href="https://fonts.googleapis.com"
+    >
+    <link
+    rel="preconnect"
+    href="https://fonts.gstatic.com"
+    >
+    <link
+    href="https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap"
+    rel="stylesheet"
+    >
+  </head>
+  <body>
+    <header>
+      <img
+        src="images/logo.png"
+        alt="logo"
+        class="logo_image"
+        >
+      <nav class="nav">
+        <div class="link-box">
+          <a class="is-active nav_link" href="#">APPLE</a>
```

Comment:
```suggestion
          <a class="is-active nav_link" href="#">Apple</a>
```
Тут текст обычный, а в css делай uppercase